game-core/src/main/java/games/strategy/triplea/attachments/RulesAttachment.java

```diff
@@ -1123,7 +1123,7 @@ private boolean checkAtWar(final PlayerID player, final Set<PlayerID> enemies, f
       }
     }
     if (count == 0) {
-      return count == found;
+      return 0 == found;
```

Comment:
The argument and the 0 should be switched

Reply:
ab12f01